src/core/bindEvents.js

```diff
@@ -40,9 +44,9 @@ function bindEvents() {
     if (facet.facetsEl) {
         this.facetWrappers.forEach(wrapper => {
             this.delegate(wrapper, facet.facetAction, `.${facet.facetClass}`, this.findChangedFacet.bind(this));
-            this.delegate(wrapper, 'change', '.' + actionChangeClass, this.extraActionsChange.bind(this));
-            this.delegate(wrapper, 'keyup', '.' + actionChangeClass, this.extraActionsChange.bind(this));
-            this.delegate(wrapper, 'click', '.' + actionBtnClass, this.extraActions.bind(this));
+            this.delegate(wrapper, 'change', '.' + (this.options.actionChangeClass  || actionChangeClass), this.extraActionsChange.bind(this));
```

Comment:
first priority should be for actionChangeClass inside facets and then this.options.actionChangeClass. same for all 3 lines